### PR TITLE
Style modifications in docusaurus docs to fix #3329

### LINKED ISSUE #3329: Style issue in Docusaurus docs
We have too much padding on menu entries in the left-hand side bar of the docs:

image

This was probably introduced by mistake in #3178.

We should restore this padding to Docusaurus' defaults - the area that needed more padding is on the right-hand side (see PR discussion).

Mentioned by @allanaaa in today's documentation meeting.

### PR DESCRIPTION
**Pull request by leoshrey**
Fixes #3329 

- Padding is now restored to Docusaurus defaults.( Corrected the mistake in #3178 as mentioned by @wetneb )

check it out

Screenshot 2020-11-26 at 11 22 31 PM

### CODE REVIEW

File: docs/src/css/custom.css [hunk 1]
```diff
@@ -38,8 +38,8 @@
 
   .menu__link--sublist {
     margin-top: 10px;
-    padding-bottom: 25px;
-    padding-top: 25px;
+    padding-bottom: 6px;
+    padding-top: 6px;
```
Comment (wetneb) [inline docs/src/css/custom.css:42]: If these are the Docusaurus defaults, could we simply remove these lines?

**Review by wetneb — COMMENTED** [review]
Thanks a lot for taking care of this!

File: docs/src/css/custom.css (hunk 1 above)
Comment (leoshrey) [reply docs/src/css/custom.css:42]: Actually, this is a great idea. I have checked it too and it is working fine.

Comment (leoshrey) [reply docs/src/css/custom.css:42]: I think we can remove.

**Comment by leoshrey** [comment]
Since this is tested, do you want me to raise another PR or edit this one?

**Comment by wetneb** [comment]
Please edit this PR directly, you should be able to do it by committing and pushing your changes on the existing branch.

**Review by wetneb — APPROVED** [review]
Thank you very much!

**Pull request merged**